src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -216,18 +216,22 @@ private void loadSheets(IDataSourceFile dataSourceFile, Cache<string, DataSheet>
             return;
          }
 
-         var mappings = dataSourceFile.Format.Parameters.OfType<MetaDataFormatParameter>().Where(p => p.ColumnName != null).Select(md => new MetaDataMappingConverter()
-         {
-            Id = md.MetaDataId,
-            Index = sheetName => md.IsColumn ? dataSourceFile.DataSheets[sheetName].RawData.GetColumnDescription(md.ColumnName).Index : -1
-         }).Union
+         var dataMappings = dataSourceFile.Format.Parameters.OfType<MetaDataFormatParameter>().Where(p => p.ColumnName != null).Select(md =>
+            new MetaDataMappingConverter()
+            {
+               Id = md.MetaDataId,
+               Index = sheetName => md.IsColumn ? dataSourceFile.DataSheets[sheetName].RawData.GetColumnDescription(md.ColumnName).Index : -1
```

Comment:
-1 again.  @abdelr and I talked about this. I don't like those magic values popping up everwhere. 
also `dataSourceFile.DataSheets[sheetName].RawData.GetColumnDescription(md.ColumnName)` type of code is everywhere. I think we need to create helper method here, maybe in the dataSourceFile to simplify all our a.b.c.d.

Reply:
yeap, we need a new issue for this :)